### PR TITLE
batch sequence output transfers to CPU

### PR DESCRIPTION
**Pull request by beingamanforever**
`Sequence.output_ids`, `output_logits` and `output_probs` construct CPU tensors from lists of GPU scalar tensors. This extracts each scalar individually, adding overhead when retrieving longer outputs. Stacking each field on the GPU before transferring it to CPU removes these individual scalar extractions

**Measured on an A10G**
> Retrieving all three fields, median of 11 repetitions after warmup:

Tokens | Current | Batched transfer
-- | -- | --
64 | 1.940 ms | 0.221 ms
512 | 17.713 ms | 1.000 ms
4,096 | 141.345 ms | 7.350 ms

</div></div></div>

### CODE REVIEW

**Comment by beingamanforever** [comment]
closes https://github.com/tiiuae/Falcon-Perception/issues/35 
Again thank you for this awesome tiny OCR model, would love to connect if possible for working on falcon or other AI research projects thank you! https://beingamanforever.github.io

**Comment by lkhphuc** [comment]
make sense, thank you.
